css/signin-signup.css

```diff
@@ -0,0 +1,141 @@
+.login-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: var(--color-grey-lighter);
+}
+
+.login {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-white);
+    border: 1px solid var(--color-grey-light);
+    border-radius: 0.4rem;
+    box-shadow: 5px 5px 5px var(--color-lighter-grey);
```

Comment:
This box-shadow can be used in many places so consider making it a variable.

Reply:
That's actually so convenient. Thanks Mahesh!